.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,50 @@
+name: Feature Request (Low Priority)
+description: Create a public Feature Request. Note that these may not be addressed as quickly as the helpdesk and that looking up account information will be difficult.
+title: "[FEATURE]: "
+labels: community, feature-request
+body:
+  - type: input
+    attributes:
+      label: Gem Name
+      description: "If your feature request is to add instrumentation support for a Ruby gem please provide the name here"
+      placeholder: mysql
+    validations:
+      required: false
+
+  - type: input
+    attributes:
+      label: Gem Version(s)
+      description: "If your feature request is to add instrumentation support for a Ruby gem please provide the version you use"
```

Comment:
Also here, "please provide the version of the above gem that you use"

Reply:
LMKWYT